src/test/java/org/jenkinsci/plugins/workflow/CpsDefaultGroovyMethodsTest.java

```diff
@@ -0,0 +1,55 @@
+package org.jenkinsci.plugins.workflow;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RunWith(Parameterized.class)
+public class CpsDefaultGroovyMethodsTest {
+    @ClassRule
+    public static JenkinsRule r = new JenkinsRule();
+    @ClassRule
+    public static BuildWatcher buildWatcher = new BuildWatcher();
+
+    private String testName;
+    private String testCode;
+    private Object testResult; // Unused
```

Comment:
Cleaner to just process the parameters from the upstream test.

Or, use this parameter, by `assert`ing that the expression equals the expected value.